lms/djangoapps/grades/models.py

```diff
@@ -338,6 +341,28 @@ def __unicode__(self):
             self.first_attempted,
         )
 
+    @classmethod
+    def prefetch(cls, course_key, users):
```

Comment:
Why was this needed?  How does it work for Grade reports today where it also needs to get all Subsection grades for batched users?